script.js

```diff
@@ -1,66 +1,462 @@
-let count = 0, itemNumber = 0;
+import { nanoid } from 'https://cdn.jsdelivr.net/npm/nanoid/nanoid.js';
 let list = [];
 
-
 //Function to add item to to-do list
-function list_additem() {
-  let input = document.getElementById("input_notes");
-
-  list.push({
-    id: count++,
-    description: input.value,
-    taskCompleted: false
-  });
-
-  //modal display on empty input
-  if (input.value.trim() === "") {
-    document.getElementById("modal_parent").style.display = "flex";
-    document.getElementById("input_notes").blur();
-    return;
-  }
-
-  let newItem = "<li><div class='list_content'><input type='checkbox' class='checkbox' id='c" + list[itemNumber].id + "'><span id = 'd" + list[itemNumber].id + "' class='description'>" + list[itemNumber].description + "</span></div><div class='buttons'><button class='edit_button'><img src='./images/icons8-edit-24.png'></button><button class='remove_button'><img src='./images/icons8-trash-can-50.png'></button></div></li>";
-
-  document.getElementById("printing_list").insertAdjacentHTML('afterbegin', newItem);
-  input.value = "";
-
-
-  let checkBox = document.getElementById("c" + list[itemNumber].id)
-  let description = document.getElementById("d" + list[itemNumber].id)
-
-  //checkbox functionalities
-  function toggleCheckedClass(){
-    if (checkBox.checked) {
-      description.classList.add("checkboxChecked");
-    }
-    else {
-      description.classList.remove("checkboxChecked");
-    }
-  }
-
-  checkBox.addEventListener("change", toggleCheckedClass)
-  
-  description.addEventListener("click", function () {
-    checkBox.checked = checkBox.checked ? false : true;
-    toggleCheckedClass();
-  })
-
-  itemNumber++;
+function list_additem(listItem) {
+	if (listItem === undefined) {
+		let input = _("input_notes");
+		let date = new Date(Date.now()).toLocaleDateString();
```

Comment:
Wouldn't just `new Date().toLocaleDateString()` work?

```suggestion
		let date = new Date().toLocaleDateString();
```